src/Core/Prefab/GlobeView.js

```diff
@@ -280,26 +272,23 @@ GlobeView.prototype.setRealisticLightingOn = function setRealisticLightingOn(val
     this.atmosphere.setRealisticOn(value);
     this.atmosphere.updateLightingPos(coSun);
 
-    this.updateMaterialUniform('lightingEnabled', value);
-    this.updateMaterialUniform('lightPosition', coSun);
+    this.updateMaterialProperty('lightingEnabled', value);
+    this.updateMaterialProperty('lightPosition', coSun);
     this.notifyChange(this.tileLayer);
 };
 
 GlobeView.prototype.setLightingPos = function setLightingPos(pos) {
     const lightingPos = pos || CoordStars.getSunPositionInScene(this.ellipsoid, new Date().getTime(), 48.85, 2.35);
 
-    this.updateMaterialUniform('lightPosition', lightingPos.clone().normalize());
+    this.updateMaterialProperty('lightPosition', lightingPos.clone().normalize());
     this.notifyChange(this.tileLayer);
 };
 
-GlobeView.prototype.updateMaterialUniform = function updateMaterialUniform(uniformName, value) {
+GlobeView.prototype.updateMaterialProperty = function updateMaterialProperty(property, value) {
```

Comment:
The fact that a material property is a uniform or a definition is an implementation detail of the material. Code base outside materials should not be aware of uniforms and defines.